rawtodigi_oneapi.h

```diff
@@ -11,7 +11,7 @@
 namespace oneapi {
   void rawtodigi(const Input *input_d, Output *output_d,
                  const uint32_t wordCounter,
-                 bool useQualityInfo, bool includeErrors, bool debug, cl::sycl::queue & stream);
+                 bool useQualityInfo, bool includeErrors, bool debug, cl::sycl::queue queue);
```

Comment:
Just to double-check, the `sycl::queue` is supposed to be passed around by value (like `cudaStream_t`)?

Reply:
Yes, according to the latest SYCL specs
> The SYCL `queue` class provides the common reference semantics (see Section 4.3.2).

which in turn say (`T` is a generic placeholder):
> `T` must be copy constructible and copy assignable on the host application and within SYCL kernel functions in the case that `T` is a valid kernel argument.  Any instance of `T` that is constructed as a copy of another instance, via either the copy constructor or copy assignment operator, must behave as-if it were the original instance  and  as-if  any  action  performed  on  it  were  also  performed  on  the  original  instance  and  if  said instance is not a host object must represent and continue to represent the same underlying OpenCL objects as the original instance where applicable.

So I understand it to mean that we can safely hold and copy aorund most SYCL objects by value.

Reply:
Ok, thanks.